### PR TITLE
Update docs with new settings

### PR DESCRIPTION
**Pull request by harshavardhana**
## Description
Update docs with new settings

## Motivation and Context
Add new settings found during testing to be optimal

## Regression
No

## How Has This Been Tested?
Using a full stack hadoop horton works cluster

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added unit tests to cover my changes.
- [ ] I have added/updated functional tests in [mint](https://github.com/minio/mint). (If yes, add `mint` PR # here: )
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by kannappanr — APPROVED** [review]
LGTM

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-compression-dist-xl.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-xl.sh | :x: [more...](#2f57df5e-ac90-4d84-b555-ab7ffdc6619f) |
| mint-large-bucket.sh | :x: [more...](#cad09c1f-276b-4a1d-a38b-e86951dcf9e1) |
| mint-dist-xl.sh | :x: [more...](#7d40beb5-7986-49ab-bc48-9093e33a1c15) |
<a name="cad09c1f-276b-4a1d-a38b-e86951dcf9e1"></a>
### 7803-fcca9fb/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.57:31358
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 89b64c99dc59:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 3 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-yyh0xd65dn5o3m",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 2673,
  "error": "RequestError: send request failed\ncaused by: Put http://72.28.97.57:31358/aws-sdk-go-test-yyh0xd65dn5o3m: dial tcp 72.28.97.57:31358: connect: no route to host",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```
<a name="2f57df5e-ac90-4d84-b555-ab7ffdc6619f"></a>
### 7803-fcca9fb/mint-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.57:30445
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 68823548ae31:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 3 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-jjgm090kwjrsce",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 3076,
  "error": "RequestError: send request failed\ncaused by: Put http://72.28.97.57:30445/aws-sdk-go-test-jjgm090kwjrsce: dial tcp 72.28.97.57:30445: connect: no route to host",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```
<a name="7d40beb5-7986-49ab-bc48-9093e33a1c15"></a>
### 7803-fcca9fb/mint-dist-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.57:30109
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp d0ad2cab68a2:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 3 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-gsdaisxztfvcnd",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 2733,
  "error": "RequestError: send request failed\ncaused by: Put http://72.28.97.57:30109/aws-sdk-go-test-gsdaisxztfvcnd: dial tcp 72.28.97.57:30109: connect: no route to host",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-compression-dist-xl.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :x: [more...](#230e1f73-1b8b-4dc1-8fd9-d5ec24709864) |
| mint-compression-fs.sh | :x: [more...](#25f67135-a031-4470-84c0-228e091ff135) |
| mint-dist-xl.sh | :x: [more...](#294f0f50-6804-4727-8085-c32f5885adf3) |
<a name="230e1f73-1b8b-4dc1-8fd9-d5ec24709864"></a>
### 7803-8d27b28/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.58:30464
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp b1aee6c786ff:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 2 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 51 seconds
(4/14) Running aws-sdk-ruby tests ... done in 16 seconds
(5/14) Running awscli tests ... done in 1 minutes and 25 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... FAILED in 2 minutes and 43 seconds
{
  "name": "mc",
  "duration": "105399",
  "function": "test_mirror_list_objects_storage_class",
  "status": "FAIL",
  "error": "/mint/run/core/mc/mc --config-dir /tmp/.mc-24003 --quiet --no-color mirror --storage-class REDUCED_REDUNDANCY /mint/data myminio/mc-test-bucket-24302  >>> \n`/mint/data/datafile-1-MB` -> `myminio/mc-test-bucket-24302/datafile-1-MB`\n`/mint/data/datafile-0-b` -> `myminio/mc-test-bucket-24302/datafile-0-b`\n`/mint/data/datafile-1-kB` -> `myminio/mc-test-bucket-24302/datafile-1-kB`\n`/mint/data/datafile-1-b` -> `myminio/mc-test-bucket-24302/datafile-1-b`\n`/mint/data/datafile-10-MB` -> `myminio/mc-test-bucket-24302/datafile-10-MB`\n`/mint/data/datafile-1.03-MB` -> `myminio/mc-test-bucket-24302/datafile-1.03-MB`\n`/mint/data/datafile-100-kB` -> `myminio/mc-test-bucket-24302/datafile-100-kB`\n`/mint/data/datafile-10-kB` -> `myminio/mc-test-bucket-24302/datafile-10-kB`\n`/mint/data/datafile-129-MB` -> `myminio/mc-test-bucket-24302/datafile-129-MB`\n`/mint/data/datafile-5-MB` -> `myminio/mc-test-bucket-24302/datafile-5-MB`\n`/mint/data/datafile-33-kB` -> `myminio/mc-test-bucket-24302/datafile-33-kB`\n`/mint/data/datafile-11-MB` -> `myminio/mc-test-bucket-24302/datafile-11-MB`\n`/mint/data/datafile-6-MB` -> `myminio/mc-test-bucket-24302/datafile-6-MB`\n`/mint/data/datafile-5243880-b` -> `myminio/mc-test-bucket-24302/datafile-5243880-b`\n`/mint/data/datafile-65-MB` -> `myminio/mc-test-bucket-24302/datafile-65-MB`\nmc: <ERROR> Failed to copy `/mint/data/datafile-129-MB`. Please reduce your request\nmc: <ERROR> Failed to copy `/mint/data/datafile-65-MB`. Please reduce your request\nmc: <ERROR> Failed to copy `/mint/data/datafile-11-MB`. Please reduce your request\nmc: <ERROR> Failed to copy `/mint/data/datafile-10-MB`. Please reduce your request\nTotal: 233.17 MiB, Transferred: 552.84 MiB, Speed: 5.25 MiB/s"
}

Executed 6 out of 14 tests successfully.
```
<a name="25f67135-a031-4470-84c0-228e091ff135"></a>
### 7803-8d27b28/mint-compression-fs.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.57:32433
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 9a4cb980d021:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 3 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-qydorugdfvu6vz",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 3049,
  "error": "RequestError: send request failed\ncaused by: Put http://72.28.97.57:32433/aws-sdk-go-test-qydorugdfvu6vz: dial tcp 72.28.97.57:32433: connect: no route to host",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```
<a name="294f0f50-6804-4727-8085-c32f5885adf3"></a>
### 7803-8d27b28/mint-dist-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.57:30878
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp ef37d8866dfb:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 2 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-dx4jvbpgeho49p",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 2406,
  "error": "RequestError: send request failed\ncaused by: Put http://72.28.97.57:30878/aws-sdk-go-test-dx4jvbpgeho49p: dial tcp 72.28.97.57:30878: connect: no route to host",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```

File: docs/bigdata/README.md [hunk 1]
```diff
@@ -40,7 +40,20 @@ Add the following optimal entries for _core-site.xml_ to configure _s3a_ with **
...
+*  _fs.s3a.path.style.access=true_
+*  _fs.s3a.block.size=256M_
+*  _fs.s3a.commiter.name=magic_
+*  _fs.s3a.committer.magic.enabled=true_
+*  _fs.s3a.committer.threads=16_
+*  _fs.s3a.connection.maximum=32_
+*  _fs.s3a.fast.upload.active.blocks=8_
+*  _fs.s3a.max.total.tasks=16_
+*  _fs.s3a.threads.core=32_
+*  _fs.s3a.threads.max=32_
```
Comment (wlan0) [inline docs/bigdata/README.md:55]: fs.s3a.ssl.enabled=true/false

Comment (harshavardhana) [reply docs/bigdata/README.md:55]: We don't need to talk about that, this is our recommended settings.

**Review by wlan0 — APPROVED** [review]

**Pull request merged**